### DISCUSSION TITLE
Could you exactly clarify how TIMING=1 for eslint is helping with caching?

### DISCUSSION THREAD

**Discussion by Guria** [vercel/turborepo#4711] [Help] [answered]
Documentation says:
> Pro Tip for caching ESLint: You can get a cacheable pretty terminal output (even for non-errors) by setting TIMING=1 variable before eslint. Learn more over in the [ESLint docs(opens in a new tab)](https://eslint.org/docs/latest/developer-guide/working-with-rules#per-rule-performance).

I have followed eslint documentation and tried to add env variable. I am just not sure how it affects the cacheability since it outputs performnce measurements for rules. Please provide more details what is stands behind this tip.

**Comment by anthonyshew** [comment, marked as answer]
It looks like this is stale and that our docs don't mention this variable anymore.

For clarity, the `TIMING` variable would not affect caching.
